speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/conformationmenu.h

```diff
@@ -0,0 +1,28 @@
+/***************************************************************************
+
+    file                 : conformation menu.h
+    created              : tuesday Mar 10 10:48:22 CET 2022
+    copyright            : (C) 2022 Maik Vink
+    email                : torcs@free.fr
+    version              : $Id: conformationmenu.h ... $
+
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef _CONFORMATIONMENU_H_
+#define _CONFORMATIONMENU_H_
+
+extern void* ConformationMenuInit(void* prevMenu, int p_saveWayVersion);
+
+#endif /* _CONFORMATIONMENU_H_ */
+
+#define RESTART 1
+#define EXIT    2
```

Comment:
also add a `typedef unsigned int ExitType;` or something like that, so it's clear that a function parameter is one of these enums instead of an actual int.